apps/admin-gui-e2e/src/e2e/admin/perun-admin.cy.js

```diff
@@ -199,6 +200,64 @@ describe('Perun admin management with role Perun admin', () => {
     });
   });
 
+  context('Blocked logins management', () => {
+    beforeEach(() => {
+      cy.get('[data-cy=admin-button]')
+        .click();
+    });
+
+    it("test display blocked logins page", () => {
+      cy.get('[data-cy=blocked-logins]')
+        .click({force: true})
+        .url()
+        .should('include', '/blocked_logins');
+    })
+
+    it('test block login', () => {
+      cy.get('[data-cy=blocked-logins]')
+        .click({force: true})
+        .intercept('**/usersManager/blockLogins**')
+        .as('blockLogin')
+        .intercept('**/usersManager/getBlockedLoginsPage**')
+        .as('getBlockedLogins')
+        .get('[data-cy=block-logins-button]')
+        .click({force: true})
+        .get('[data-cy=logins-input]')
+        .type("loginToBlock", {force: true})
```

Comment:
```suggestion
        .type(loginToBlock, {force: true})
```
Use a constant instead, please